pdns/syncres.cc

```diff
@@ -795,11 +795,12 @@ bool SyncRes::doCNAMECacheCheck(const DNSName &qname, const QType &qtype, vector
   return false;
 }
 
-static const DNSName getLastLabel(const DNSName& qname)
-{
-  DNSName ret(qname);
-  ret.trimToLabels(1);
-  return ret;
+static inline void addTTLModifiedRecords(const vector<DNSRecord>& records, const uint32_t ttl, vector<DNSRecord>& ret) {
```

Comment:
I would advise to drop the `inline` keyword. The compiler will try to inline when it makes sense anyway, but with the keyword it will generate a warning if the inlining doesn't happen.